Code/Framework/AzFramework/AzFramework/Physics/Character.cpp

```diff
@@ -127,6 +128,9 @@ namespace Physics
                         ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
                     ->DataElement(AZ::Edit::UIHandlers::Default, &CharacterConfiguration::m_colliderTag,
                         "Collider Tag", "Used to identify the collider associated with the character controller")
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &CharacterConfiguration::m_applyMoveOnPhysicsTick,
+                        "Apply Move On Tick", "Requests to add velocity will be accumulated and applied once on the physics pre-simulation tick "
```

Comment:
NP: Some extra white space at the end

Reply:
It's because this and the string below concatenate and without the whitespace it'd be "...pre-simulation tickIf unticked.."

Reply:
Clearly it's too early in the day for me!